vst/SfizzVstProcessor.cpp

```diff
@@ -105,21 +112,27 @@ tresult PLUGIN_API SfizzVstProcessor::canProcessSampleSize(int32 symbolicSampleS
 
 tresult PLUGIN_API SfizzVstProcessor::setActive(TBool state)
 {
-    stopBackgroundWork();
-    _synth.reset();
+    sfz::Sfizz* synth = _synth.get();
 
-    if (state) {
-        fprintf(stderr, "[Sfizz] new synth\n");
-        sfz::Sfizz* synth = new sfz::Sfizz;
+    if (!synth) {
```

Comment:
Why create the synth here, not just return a kResultFalse?
Does this fix bad host behavior that you observed?